homeassistant/components/sensor/teksavvy.py

```diff
@@ -31,11 +32,11 @@
 REQUEST_TIMEOUT = 5  # seconds
 
 SENSOR_TYPES = {
-    'usage': ['Usage', PERCENT, 'mdi:percent'],
+    'usage': ['Usage Ratio', PERCENT, 'mdi:percent'],
     'usage_gb': ['Usage', GIGABYTES, 'mdi:download'],
     'limit': ['Data limit', GIGABYTES, 'mdi:download'],
     'onpeak_download': ['On Peak Download', GIGABYTES, 'mdi:download'],
-    'onpeak_upload': ['On Peak Upload ', GIGABYTES, 'mdi:upload'],
+    'onpeak_upload': ['On Peak Upload', GIGABYTES, 'mdi:upload'],
```

Comment:
This and line 35 would be breaking changes?

How do I ensure that makes it into the release notes?

Reply:
Notably `sensor.teksavvy_on_peak_upload_` (which is incorrect with the trailing underscore) becomes `sensor.teksavvy_on_peak_upload`

Additionally with both the percentage and GB values both being named `usage` you would get a non-deterministic result in tests for `sensor.teksavvy_usage` and `sensor.teksavvy_usage_2`.
Renaming to `Usage Ratio` makes it very deterministic.

Reply:
Add a section to your PR description and I'll add the label "Breaking Change"

Reply:
Added a description, thanks!